Schedule_app/views.py

```diff
@@ -94,7 +95,7 @@ def seats(response):
             filter(name__icontains='cinema{}'.format(data['cinema'])) \
             .filter(name__icontains='hall{}'.format(data['hall'])). \
             order_by('id').values()
-        reserved_seats = BookTicket.objects.filter(schedule=schedule[0]['id']).values()
+        reserved_seats = BookTicket.objects.filter(schedule=schedule[0]['id'], book_confirmed=True).values()
```

Comment:
Below, in the for loop you can loop directly over the values, instead of using the index:
for res in reserved_seats:
    if seat['id'] in res['seats']:
        seat['available'] = False